terraform/envs/dev/Makefile

```diff
@@ -12,10 +16,10 @@ apply:
 	terraform apply --backup=-
 
 retrieve-vars:
-	aws s3 cp s3://org-humancellatlas-upload-infra/terraform/envs/dev/terraform.tfvars terraform.tfvars
+	aws s3 cp $(S3_TFVARS_FILE) terraform.tfvars
```

Comment:
Seems like this could be more secure if this was stored in Secrets Manager. Here's some sample code [from the logs repo](https://github.com/HumanCellAtlas/logs/blob/577a37c4b2f32cefe199d77b74c483c9db1e1fff/Makefile#L7-L12).